### DISCUSSION TITLE
Difference between information content entity (IBE) and information content artifact (IBA)?

### DISCUSSION THREAD

**Discussion by alanruttenberg** [CommonCoreOntology/CommonCoreOntologies#405] [Announcements]
Setting aside the ICE vs IBE discussion, what's the intended difference IBA and IBE? Other than IBA, the only child of IBE is *Document field*. I thought perhaps the intended difference was whole/part, but both are objects. *Information Line* seems like a similar sort of part and it is an IBA.

**Comment by rorudn** [comment]
IBEs correspond to a value expressed by a single primitive type (i.e. integer, string, boolean, double) where IBA are complexes of values.

**Comment by alanruttenberg** [comment]
If that's the intention, then I think it needs to say so, as I don't think it conveys that currently. But note that an information line could contain a single value, and a data field could contain value of a complex type. Some of the subs of designative information entities seem to be simple. 

I don't think that's going to be worth changing until we're resolved the ICE/IBE discussion. FWIW I've left the representation of primitive types to OWL's datatypes and not added them as classes. I will soon be needing _formats_ of data values, to describe parts of documents.
